### DISCUSSION TITLE
Show `Fixed Speed` when adjust `Scroll Speed`

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#20102] [Ideas] [answered]
One of my mania friends asked me to show fix speed or not ms in `scroll speed` settings
![image](https://user-images.githubusercontent.com/34775378/188154669-3ba65a09-fc7d-4dc2-ac7e-c8f538460830.png)
like this

https://user-images.githubusercontent.com/34775378/188154770-a010fac5-07ce-4e0b-8781-2a22fdbeb809.mp4

**Comment by peppy** [comment]
This is already displayed in the HUD overlay during gameplay adjustments, so if you're adding this make sure it uses the same calculation and display formatting logic.

**Reply by cdwcgt** [reply]
yes.
It is the same logic with overlay display

**Reply by peppy** [reply]
On a quick check the formatting looks different (back to front) so make sure that matches.

**Reply by cdwcgt** [reply]
```C#
        private class ManiaScorllSlider : OsuSliderBar<double>
        {
            public override LocalisableString TooltipText => $"{Current.Value:N0} ms ({(int)Math.Round(DrawableManiaRuleset.MAX_TIME_RANGE / Current.Value)})";
        }
```
The calculation code is basically copied, which can be said to be a very small change

**Reply by peppy** [reply]
I'm talking about the display method, which is different:

https://github.com/ppy/osu/blob/f8830c6850128456266c82de83273204f8b74ac0/osu.Game.Rulesets.Mania/Configuration/ManiaRulesetConfigManager.cs#L36

it should probably match in format.

**Reply by cdwcgt** [reply]
I don't quite understand
The display method here only display at gameplay
So I copied the fixed speed in slider tooltip

you mean show fixed speed in front of ms?

like `26 (440 ms)`?

**Reply by peppy** [reply]
Yep, exactly. Just have both display the same way for consistency.

**Reply by cdwcgt** [reply]
ok, that is easy

do I open a pr after the modification is done?

**Reply by peppy** [reply]
Yep. For small changes like this you can probably skip making an issue and open a PR directly (might be faster), but up to you either works.

**Comment by cdwcgt** [comment, marked as answer]
https://github.com/ppy/osu/pull/20190
